app/models/goal.py

```diff
@@ -1,5 +1,12 @@
 from app import db
-
-
 class Goal(db.Model):
-    goal_id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String, nullable=False)
+    tasks = db.relationship('Task', backref='goal', lazy=True)
+
+    def goal_response_body_dict(self):
+        response_goal = {
+            "id": self.id,
+            "title": self.title
+        }
+        return response_goal
```

Comment:
Style: Since you immediately return this dictionary you don't need to assign it to a variable:

```suggestion
        response_goal = {
            "id": self.id,
            "title": self.title
        }
        return response_goal
```
```suggestion
        return {
            "id": self.id,
            "title": self.title
        }
```